documentation/how-to/temporal-filtering.en.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Temporal filtering
+---
+
+# Temporal filtering
+
+When QField consumes projects containing one or more layers with active temporal settings, a temporal filtering indicator located within the side dashboard appears next to the map theme combo box. A green clock icon indicates temporal properties are filtering features drawn on the map canvas is active, while a gray clock icon indicates the filtering is turned off.
+
+!![](../assets/images/temporal-properties.png)
+
+## Adjusting temporal filtering
+
+When clicking on the side dashboard's temporal filtering indicator, a temporal properties panel pops up. From there, you can enable the temporal filter and set the temporal range's beginning and end time. 
```

Comment:
> side dashboard

Is this what we officially use? Side panel? Side drawer? Legend panel? No idea, just to make sure we use the same terminology across docs.

Reply:
@suricactus , we use dashboard elsewhere (e.g. the map themes page).